src/_locales/ja/messages.json

```diff
@@ -69,7 +69,7 @@
     "message": "CaretTabにようこそ!"
   },
   "welcomeUpdated": {
-    "message": "CaretTabのアップデートがあります．"
+    "message": "アップデート履歴"
```

Comment:
I see your fix seems to translate back to "Update History" whereas the old version translates back to "There is an update of CaretTab" which matches the English translation better. Can you double check this? Keep in mind this sentence is followed by the current version number. So in English it says "CaretTab has been updated to" followed by "v2.x.x". Thanks!

Reply:
Sorry, I confused "welcomeUpdated" with "latestUpdate". Your review is correct. The message of "welcomeUpdated" should be kept with the old version. Can I make another pull request which is revised this issue?
(When I checked the Japanese translation of CaretTab, I thought the Japanese message of "最新のアップデート(Latest Update)" should be changed "アップデート履歴(Update History)" since the area of "latestUpdate" shows the history of versions.)
Thanks.